### PR TITLE
Added initial snippets

### PR DESCRIPTION
**Pull request by ghost**
Added initial snippets for:
- Create, List, Get, Update, Delete Endpoint Lambda
- Tests for the above
- Unroll, it, describe for jest
- Arrow Function

### CODE REVIEW

**Comment by shishiranshuman** [comment]
Change the destination to a version branch.

File: README.md [hunk 1]
```diff
@@ -1 +1,41 @@
...
+| `delete`            | Snippet for Delete Endpoint Lambda         |
+| `testcreate`        | Snippet for testing Create Endpoint Lambda |
+| `testlist`          | Snippet for testing List Endpoint Lambda   |
+| `testget`           | Snippet for testing Get Endpoint Lambda    |
+| `testupdate`        | Snippet for testing Update Endpoint Lambda |
+| `testdelete`        | Snippet for testing Delete Endpoint Lambda |
+| `arrow`             | Snippet for Arrow Function                 |
+| `it`                | Snippet for it (testing)                   |
+| `unroll`            | Snippet for unroll (testing)               |
+| `describe`          | Snippet for describe (testing)             |
```
Comment (shishiranshuman) [inline README.md]: These snippets are already added in the `snips-react`. Do we really need them here as well?

File: README.md [hunk 2]
```diff
@@ -1 +1,41 @@
...
+3. From the command palette `Ctrl`-`Shift`-`X` (Windows, Linux) or `Cmd`-`Shift`-`X` (OSX)
+4. Select `Install Extension`
+5. Choose the extension
+6. Reload Visual Studio Code
+
+
+##Contribute
+
+This is an open source project and if you want to contribute, feel free to create issue in our [repository](https://github.com/causecode/snips-serverless) or create pull requests.
+
```
Comment (shishiranshuman) [inline README.md:44]: Add a block for the `License`

File: snippets/typescript.json [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+            "}, [ // tslint:disable-next-line",
+            "\t[${0}],",
+            "\t[],",
+            "\t[],",
+            "]);\n"
+        ],
+        "description": "Snippet for unroll (testing)"
+    },
+
+    "it for jest": {
```
Comment (adityamedhe-cc) [inline snippets/typescript.json]: `Test case for Jest`. Also this is present in the React extension. Do we need it here

File: README.md (hunk 2 above)
Comment (ghost) [reply README.md:44]: Done.

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: As the three extensions [`snips-react`, `snips-sequelize` and `snips-serverless`] are meant to be maintained separately, it only makes sense to keep the common snippets in all of them as one working on **react** (using `snips-react`) should not install `snips-sequelize` extension just to get the testing snippets. Leaving this as-is.

File: snippets/typescript.json (hunk 3 above)
Comment (ghost) [reply snippets/typescript.json]: Yes we do, as the three extensions [`snips-react`, `snips-sequelize` and `snips-serverless`] are meant to be maintained separately, it only makes sense to keep the common snippets in all of them as one working on **react** (using `snips-react`) should not install `snips-sequelize` extension just to get the testing snippets. Leaving this as-is.

**Comment by ghost** [comment]
Added new snippets and fixed suggested changes.

**Pull request merged**